doc/content/en/docs/++version++/Specification/_index.md

```diff
@@ -259,13 +259,29 @@ Complex types (`record`, `enum`, `array`, `map`, `fixed`) have no namespace, but
 
 A schema or protocol may not contain multiple definitions of a fullname. Further, a name must be defined before it is used ("before" in the depth-first, left-to-right traversal of the JSON parse tree, where the types attribute of a protocol is always deemed to come "before" the messages attribute.)
 
-### Aliases
+### Aliases {#aliases}
```

Comment:
What is this ?
Is it something for the url `#fragment` ? Because if it is then IMO there is no need of it - it already works: https://avro.apache.org/docs/1.11.1/specification/#aliases

Reply:
Good point. I copied this from other places where I saw a link being used. But I guess it's not needed in all cases.

Reply:
I'll remove it, and also the explicit anchor `{#names}`